lib/exq/enqueue_api.ex

```diff
@@ -18,16 +19,23 @@ defmodule Exq.Enqueuer.EnqueueApi do
         * `queue` - Name of queue to use
         * `worker` - Worker module to target
         * `args` - Array of args to send to worker
+        * `options` - job options, for example [max_retries:  `Integer`]
 
       Returns:
       * `{:ok, jid}` if the job was enqueued successfully, with `jid` = Job ID.
       * `{:error, reason}` if there was an error enqueueing job
       """
-      def enqueue(pid, queue, worker, args) do
-        GenServer.call(pid, {:enqueue, queue, worker, args}, Config.get(:genserver_timeout))
+      def enqueue(pid, queue, worker, args), do: enqueue(pid, queue, worker, args, @default_options)
+
+      def enqueue(pid, from, queue, worker, args) when is_pid(from) do
+        enqueue(pid, from, queue, worker, args, @default_options)
+      end
+      def enqueue(pid, queue, worker, args, options) do
```

Comment:
1. Can't using `options \\[]` it will cause confilcts.
```ex
iex(1)> defmodule Test do
...(1)>   def sum(a,b\\0), do: a+b
...(1)>   def sum(a,b,c\\0), do: a+b+c
...(1)> end
** (CompileError) iex:7: def sum/3 defaults conflicts with def sum/2
```
2. I will create new PR for this. (something wrong with rebase :( )